minetest.conf.example

```diff
@@ -408,6 +408,13 @@
 #max_block_send_distance = 10
 #    From how far blocks are generated for clients, stated in mapblocks (16 nodes)
 #max_block_generate_distance = 6
+#    Where the map generator stops.
+#    Please note:
+#      * Limited to 31000 (setting above has no effect)
+#      * The map generator works in blocks of MapBlocks of 80x80x80 nodes
```

Comment:
Perhaps formulate this "The map generator works in blocks of MapBlocks (80x80x80 nodes)" ??